### PR TITLE
Print generated VC from CLI command

### PR DESCRIPTION
**Pull request by kziemianek**
resolves: P-177

* Prints out generated VC.
* `perform_trusted_operation` allows to specify the type of the `TrustedOpResult::Ok` variant, so no additional decoding is needed.
* fixes `aes_decrypt` function - result of `open_in_place` should be returned as it doesn't containt the tag

### CODE REVIEW

**Comment by wli-pro** [comment]
I wasn't sure now how did my test before (without this commit in this PR) even work: https://github.com/litentry/litentry-parachain/pull/2248 (see the cli test results in the PR...)

**Comment by wli-pro** [comment]
But this would be very helpful for testing, before this I used to need to get the VC from explorer

**Comment by kziemianek** [comment]
> I wasn't sure now how did my test before (without this commit in this PR) even work: #2248 (see the cli test results in the PR...)

You can find VC content in worker logs.
As a CLI user you probably won't have access to logs + production build doesn't log such information.

**Review by wli-pro — APPROVED** [review]

**Review by Kailai-Wang — APPROVED** [review]
Looks good, thanks!

Integirtee might be interested in `perform_trusted_operation` change too

File: tee-worker/cli/src/trusted_base_cli/commands/litentry/link_identity.rs [hunk 1]
```diff
@@ -79,6 +77,7 @@ impl LinkIdentityCommand {
 		)
 		.sign(&KeyPair::Sr25519(Box::new(alice)), nonce, &mrenclave, &shard)
 		.into_trusted_operation(trusted_cli.direct);
-		Ok(perform_trusted_operation(cli, trusted_cli, &top).map(|_| CliResultOk::None)?)
+		//todo proper generic type
```
Comment (kziemianek) [inline tee-worker/cli/src/trusted_base_cli/commands/litentry/link_identity.rs]: I'll remove this comment

**Review by BillyWooo — APPROVED** [review]
LGTM

File: tee-worker/sidechain/rpc-handler/src/direct_top_pool_api.rs [hunk 2]
```diff
@@ -101,9 +101,10 @@ where
 
 		while let Ok(response) = receiver.try_recv() {
 			if let Some(Ok(response)) = response {
+				// response is already encoded
 				let json_value = RpcReturnValue {
 					do_watch: false,
-					value: response.encode(),
+					value: response,
```
Comment (kziemianek) [inline tee-worker/sidechain/rpc-handler/src/direct_top_pool_api.rs:107]: @felixfaisal fyi

**Pull request merged**